src/test/Makefile

```diff
@@ -296,6 +296,7 @@ pid_unittest_SRC :=  \
 		$(USER_DIR)/drivers/accgyro/gyro_sync.c \
 		$(USER_DIR)/flight/pid.c \
 		$(USER_DIR)/pg/pg.c \
+		$(USER_DIR)/build/debug.c \
```

Comment:
I don't think you need that here - the defines required to write debug information are all in `debug.h`.

Reply:
This was added so that the unit test would link successfully.

Reply:
@mikeller any issue with linking debug.c? That seems more robust in case the debug implementation changes?

Reply:
In my opinion, for unit tests, everything that isn't the object under test should be stubbed out if this is reasonably possible.